download/doc.go

```diff
@@ -0,0 +1,4 @@
+/*
+Package download helps download releases of binaries.
```

Comment:
perhaps we can cleanup these one line doc.go's whilst we are changing things.

This can be a package comment in the downloader.go 

(the doc.go thing is only needed for rlly long package docs)